packages/bitcore-wallet-service/src/lib/pushnotificationsservice.ts

```diff
@@ -451,16 +451,30 @@ export class PushNotificationsService {
       bit: 'bits',
       bch: 'BCH',
       eth: 'ETH',
+      matic: 'MATIC',
       xrp: 'XRP',
       doge: 'DOGE',
       ltc: 'LTC',
-      usdc: 'USDC',
-      pax: 'PAX',
-      gusd: 'GUSD',
-      busd: 'BUSD',
-      wbtc: 'WBTC',
-      dai: 'DAI',
-      shib: 'SHIB'
+      matic_e: 'MATIC',
```

Comment:
There is no need for adding new keys here

Reply:
Good point. I made the adjustment in my latest commit.